From: Eric Van Hensbergen <ericvh@gmail.com>
Subject: [PATCH 003/111] net/9p: fix virtio transport to correctly update status on connect
Git-commit: 562ada612058133a5483c68a73605f3c5f42fffe
Patch-Mainline: 2.6.33
References: FATE#311639

    The 9p virtio transport was not updating its connection status
    correctly preventing it from being able to mount the server.
	upstream commit: 562ada612058133a5483c68a73605f3c5f42fffe

    Signed-off-by: Eric Van Hensbergen <ericvh@gmail.com>
Signed-off-by: Harsh Prateek Bora <harsh@linux.vnet.ibm.com>
Acked-by: Bruce Rogers <brogers@suse.com>
---
 net/9p/trans_virtio.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/net/9p/trans_virtio.c b/net/9p/trans_virtio.c
index ea1e3da..67c4bc7 100644
--- a/net/9p/trans_virtio.c
+++ b/net/9p/trans_virtio.c
@@ -311,6 +311,7 @@ p9_virtio_create(struct p9_client *client, const char *devname, char *args)
 	}
 
 	client->trans = (void *)chan;
+	client->status = Connected;
 	chan->client = client;
 
 	return 0;
-- 
1.7.1.1

